### ISSUE TITLE
My appNav.js components is not working what to do?

### ISSUE THREAD

**Issue by daiky00** [ramsaylanier/WordpressExpress#22]
Hi, 

I am getting the errors below after following the documentation you have so far. What can I do in this case? Thank you in advanced

![image](https://cloud.githubusercontent.com/assets/15005162/18536688/1f15a4e2-7acd-11e6-92d1-a66eda32fe1a.png)

`TypeError: Cannot read property 'items' of null
AppNav.render
C:/webapps/htdocs/wordpress/wp-app/app/components/nav/_AppNav.js
AppNav.tryRender
http://localhost:3000/main.js:721:2
AppNav.proxiedMethod
http://localhost:3000/main.js:6295:2
ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext
http://localhost:3000/main.js:6601:2
ReactCompositeComponentWrapper._renderValidatedComponent
http://localhost:3000/main.js:6601:2
ReactCompositeComponentWrapper.performInitialMount
http://localhost:3000/main.js:6601:2
ReactCompositeComponentWrapper.mountComponent
http://localhost:3000/main.js:6601:2
Object.mountComponent
http://localhost:3000/main.js:1321:2
Object.updateChildren
http://localhost:3000/main.js:6583:2
ReactDOMComponent._reconcilerUpdateChildren
http://localhost:3000/main.js:6757:2`

**Comment by ramsaylanier** [comment]
You have created menu items using the WordPress backend, correct? My guess is that maybe the menu slug isn't correct. I'll take a deeper look tomorrow.

**Comment by daiky00** [comment]
Hi,
 yes that's correct I create a menu using two pages: post and homepage. Any advice will be much appreciated.

**Comment by ramsaylanier** [comment]
What does your header.js component look like? My second guess is that your menu doesn't have the right slug.

Mine looks like this:

<img width="1041" alt="screen shot 2016-09-15 at 12 10 10 am" src="https://cloud.githubusercontent.com/assets/2359852/18538159/e9f8576c-7ad8-11e6-8112-43de3ce637d7.png">

Notice `menu="primary-navigation"`.  primary-navigation is the slug of the menu. Make sure your menu has that slug, or change it to whatever the slug of your current menu is. 

Basically, GraphQL isn't finding your menu, therefore in AppNav when it tries to map through each of the menu items you get an error (because menu items don't exist).

**Comment by ramsaylanier** [comment]
Lastly, here  is the documentation for WordExpressMenu component: https://github.com/ramsaylanier/WordExpressComponents#wordexpressmenu

**Comment by daiky00** [comment]
Thank You. that fixed the problem is all good now :)
